src/openassetio-python/package/openassetio/pluginSystem/PythonPluginSystemManagerImplementationFactory.py

```diff
@@ -37,48 +37,97 @@ class PythonPluginSystemManagerImplementationFactory(ManagerImplementationFactor
     derived plugins. Not usually used directly by a @ref host, which
     instead uses the @fqref{hostApi.ManagerFactory} "ManagerFactory".
 
+    The factory loads plugins from exposed `openassetio.manager_plugin`
+    entry points, as well as additional plugins found under paths
+    specified in the `OPENASSETIO_PLUGIN_PATH` env var.
+
     @envvar **OPENASSETIO_PLUGIN_PATH** *str* A PATH-style list of
-    directories to search for @ref openassetio.pluginSystem.PythonPluginSystemManagerPlugin
-    based plugins. It uses the platform-native delimiter.  Searched left
-    to right.
+    directories to search for @ref
+    openassetio.pluginSystem.PythonPluginSystemManagerPlugin based
+    plugins. It uses the platform-native delimiter.  Searched left to
+    right. Plugins found here will take precedence over those discovered
+    through package entry points. Note: this search path is entirely
+    independent of **PYTHONPATH** and doesn't not need to be a subset of
+    those paths. This allows OpenAssetIO plugins to be managed entirely
+    independent of the python runtime if desired, and masked from
+    `import` statements.
+
+    @envvar **OPENASSETIO_DISABLE_ENTRYPOINTS_PLUGINS** when set,
+    disables entry point based plugin discovery. This can be useful if
+    it is not in use, to avoid unnecessary filesystem access during
+    library initialization. **OPENASSETIO_PLUGIN_PATH** plugins take
+    precedence over any entry point based ones.
     """
 
     ## The Environment Variable to read the plug-in search path from
     kPluginEnvVar = "OPENASSETIO_PLUGIN_PATH"
+    ## The Environment Variable to control the discovery of entry point based plugins
+    kDisableEntryPointsEnvVar = "OPENASSETIO_DISABLE_ENTRYPOINTS_PLUGINS"
+
+    ## The name of the ManagerPlugin entry point for entry point
+    ## discovered plugins.
+    kPackageEntryPointName = "openassetio.manager_plugin"
```

Comment:
It's more the group name than an entry point name. See https://packaging.python.org/en/latest/specifications/entry-points/#data-model.

Reply:
Ahh yes, thanks, renamed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/768/commits/029267c8474d54ff04c4dfa4c9f1af3c5935a4bb).